task_sdk/src/airflow/sdk/definitions/mixins.py

```diff
@@ -0,0 +1,122 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import annotations
+
+from abc import abstractmethod
+from collections.abc import Iterable, Sequence
+from typing import TYPE_CHECKING, Any
+
+from ..types import NOTSET, ArgNotSet
+
+if TYPE_CHECKING:
+    from .baseoperator import BaseOperator
+    from .edges import EdgeModifier
```

Comment:
Relative imports for some reason?

Reply:
Not a good one, that's just how I used it in my poc code and I hadn't tidied these up yet. (I do kind of like `.x` imports personal when `x` is just a single level, but I just haven't fixed this yet)